.goreleaser.yaml

```diff
@@ -16,6 +16,13 @@ builds:
         - CGO_ENABLED=0
       # We want our builds to be reproducible, so we use the commit time as timestamps
       mod_timestamp: '{{ .CommitTimestamp }}'
+      ldflags:
+        - -s -w
+        - -X {{.ModulePath}}/pkg/version.application={{.Binary}}
+        - -X {{.ModulePath}}/pkg/version.version={{.Version}}
+        - -X {{.ModulePath}}/pkg/version.revision={{.FullCommit}}
+        - -X {{.ModulePath}}/pkg/version.branch={{.Branch}}
+        - -X {{.ModulePath}}/pkg/version.buildDate={{.Date}}
```

Comment:
I'm afraid that this would either duplicate the git information or make the release non-reproduceable.  I'd argue to simply remove this field or make it clearer in the naming that this is the release date and not in which second the CI/CD pipeline finished.

Reply:
I renamed it to represent the release date.